src/components/ItemDetails/ItemDetails.jsx

```diff
@@ -0,0 +1,15 @@
+import React from 'react';
+import style from './ItemDetail.css';
+
+export default function ItemDetails({ detailItem }) {
+  const { name, sprites, effect_entries } = detailItem;
```

Comment:
We could also spread detail Item in as a prop in the parent and de-structure it on line 4